microsetta_interface/templates/new_results_page.jinja2

```diff
@@ -1306,358 +1306,243 @@
+        drawIndicatorLine(angle, radius * 0.25, radius * 0.45, 5);
+    }
+
+    if (diversityInSampleValue >= 7.0 && diversityInSampleValue < highestDiversityScore) {
+        const angle = scale(diversityInSampleValue);
+        drawIndicatorLine(angle, radius * 0.45, radius * 0.25, -35);
+    }
+
+    drawRotatedLine(svg, 20, -radius + 106.5, 43.5, 56.921, "#006A96", 2);
+    drawRotatedLine(svg, 110, -radius + 194, -77, 56.921, "#006A96", 2);
+
+    svg.append("text")
+        .attr("x", -radius - 50)
+        .attr("y", 145)
+        .style("fill", "#747678")
+        .style("font-size", "14px")
+        .style("line-height", "24px")
+        .style("font-family", "Mulish")
+        .style("font-weight", "400")
+        .style("font-style", "normal")
+        .text("{{ _('Lowest in our dataset') }}");
+
+    svg.append("text")
+        .attr("x", -radius - 50)
+        .attr("y", 127.5)
+        .style("fill", "#006A96")
+        .style("font-size", "18px")
+        .style("font-weight", "700")
+        .style("line-height", "24px")
+        .text("2.40");
+
+    svg.append("text")
+        .attr("x", -radius + 67.5)
+        .attr("y", 100)
+        .style("fill", "#006A96")
+        .style("text-align", "center")
+        .style("font-size", "16px")
+        .style("font-style", "normal")
+        .style("font-weight", "700")
+        .style("line-height", "26px")
+        .text("{{ _('Diversity Value') }}");
+
+    svg.append("text")
+        .attr("x", radius - 42.5)
+        .attr("y", 145)
+        .style("fill", "#747678")
+        .style("font-size", "14px")
+        .style("line-height", "24px")
+        .style("font-family", "Mulish")
+        .style("font-weight", "400")
+        .style("font-style", "normal")
+        .text("{{ _('Highest in our dataset') }}");
+
+    svg.append("text")
+        .attr("x", radius - 42.5)
+        .attr("y", 127.5)
+        .style("fill", "#006A96")
+        .style("font-size", "18px")
+        .style("font-weight", "700")
+        .style("line-height", "24px")
+        .text("7.25");
+
+    svg.append("text")
+        .attr("x", 172.5)
+        .attr("y", -radius + 38)
+        .style("fill", "#747678")
+        .style("text-anchor", "middle")
+        .style("font-size", "14px")
+        .style("line-height", "24px")
+        .style("font-family", "Mulish")
+        .style("font-weight", "400")
+        .style("font-style", "normal")
+        .text("{{ _('Average in our dataset') }}");
+
+    svg.append("text")
+        .attr("x", 55)
+        .attr("y", -radius + 40)
+        .style("fill", "#006A96")
+        .style("font-size", "18px")
+        .style("font-weight", "700")
+        .style("line-height", "24px")
+        .text("5.31");
+
+    svg.append("text")
+        .attr("x", 110)
+        .attr("y", -radius + 185)
+        .style("fill", "#006A96")
+        .style("font-size", "18px")
+        .style("font-weight", "700")
+        .style("line-height", "24px")
+        .text("6.30");
```

Comment:
The Hadza score varies by dataset - it's 6.30 for WGS and 5.20 for 16S. Please use the existing code that determines which dataset the sample report is referencing to determine the appropriate score. The below text, related to the Hadza, also needs to be positioned appropriately depending on which dataset is being used.